### PR TITLE
feat(post): Deis v1.13.0 LTS release

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+title: "Deis v1.13.0 LTS - Long Term Support"
+author: bacongobbler
+meta:
+  - name: description
+    content: Deis release v1.13.0
+  - name: keywords
+    content: deis, release, v1.13.0, stable, production, production-ready, paas, private paas, heroku, github, docker, coreos, enterprise
+---
+
+Deis v1.13.0 LTS (Long-Term Support) is the final feature release. We will continue to patch bugs and accept pull requests, however any new features should be applied to [Deis Workflow](https://deis.com/); the successor to Deis.
```
Comment (jackfrancis) [inline _posts/2016-04-06-deis-1-13-lts.md]: might consider putting "Deis v1.13.0 LTS (Long-Term Support)" in bold to help it stand out from the description text.

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Deis v1.13.0 LTS bumps CoreOS to 899.13.0, updates the system's containers to Alpine 3.3, removes the scheduler technology previews, bumps the Heroku buildpacks to the latest stable versions, adds a healthcheck url `/healthz` to the controller, and allows logspout to re-discover the logger when it jumps to another control plane node.
```
Comment (jackfrancis) [inline _posts/2016-04-06-deis-1-13-lts.md]: ditto "Deis v1.13.0 LTS"

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## What is Deis?
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide a lightweight PaaS with a Heroku-inspired workflow.
+
+## v1.13.0 Summary
```
Comment (arschles) [inline _posts/2016-04-06-deis-1-13-lts.md]: Is it possible that someone could confuse `v1.13.0` with `v1.13.0 LTS`? Seems like that could cause confusion, but I could be overthinking...

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+ - **deis-controller** now has a health check view at `/healthz`
+ - There are graceful shutdown scripts for Ceph and Ceph-less nodes. See http://docs.deis.io/en/latest/managing_deis/add_remove_host/
+ - **deis-controller** returns a 409 when cancelling a user account that has applications
+
+### Improvements
+
+ - **deis-router** now makes vhost_traffic_status_zone configurable
+ - **deis-logspout** now continuously re-discovers the logger's connection information, should it be re-scheduled onto another host
+ - **deis-controller** prevents overlapping config:set operations
+ - **deis-client** only deletes ~/.deis/client.json if you're cancelling your own account
```
Comment (jackfrancis) [inline _posts/2016-04-06-deis-1-13-lts.md]: consider expressing the filepath as `~/.deis/client.json` (i.e., surrounded by backticks)

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+ - **deis-client** responds to `deis version`, `deis -v`, `deis --version` and `deis help version`
+ - **deis-controller** now has a health check view at `/healthz`
+ - There are graceful shutdown scripts for Ceph and Ceph-less nodes. See http://docs.deis.io/en/latest/managing_deis/add_remove_host/
+ - **deis-controller** returns a 409 when cancelling a user account that has applications
+
+### Improvements
+
+ - **deis-router** now makes vhost_traffic_status_zone configurable
+ - **deis-logspout** now continuously re-discovers the logger's connection information, should it be re-scheduled onto another host
+ - **deis-controller** prevents overlapping config:set operations
```
Comment (jackfrancis) [inline _posts/2016-04-06-deis-1-13-lts.md]: consider expressing the `config:set` thusly

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+### New Features
+
+ - **deis-client** responds to `deis version`, `deis -v`, `deis --version` and `deis help version`
+ - **deis-controller** now has a health check view at `/healthz`
+ - There are graceful shutdown scripts for Ceph and Ceph-less nodes. See http://docs.deis.io/en/latest/managing_deis/add_remove_host/
+ - **deis-controller** returns a 409 when cancelling a user account that has applications
+
+### Improvements
+
+ - **deis-router** now makes vhost_traffic_status_zone configurable
```
Comment (arschles) [inline _posts/2016-04-06-deis-1-13-lts.md]: can you put vhost_traffic_status_zone inside backticks?

File: _posts/2016-04-06-deis-1-13-lts.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+ - **deis-logspout** now continuously re-discovers the logger's connection information, should it be re-scheduled onto another host
+ - **deis-controller** prevents overlapping config:set operations
+ - **deis-client** only deletes ~/.deis/client.json if you're cancelling your own account
+ - **deis-controller** fixed a bug where single quotes could not be used in an environment variable
+ - **deis-builder** can be used from within a proxied environment
+ - **deis-database** exits if it is unable to check for existing backups in Ceph
+ - Linode's provisioning scripts now allows for cluster expansion
```
Comment (arschles) [inline _posts/2016-04-06-deis-1-13-lts.md]: `scripts` should be singular

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 1 above)
Comment (bacongobbler) [inline _posts/2016-04-06-deis-1-13-lts.md]: I've never liked bold styling in regular text unless you were trying to emphasize something. We've never done that before in our previous release notes.

**Comment by arschles** [comment]
@bacongobbler left a bunch of small nitpicks, overall looks good. Since almost all of my comments are suggestions or half commentary, I'm LGTMing now.

**Comment by jackfrancis** [comment]
I didn't see anything egregious, added some notes for your consideration, otherwise LGTM.

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 3 above)
Comment (bacongobbler) [inline _posts/2016-04-06-deis-1-13-lts.md]: Yeah I could be a little more consistent with the naming scheme.

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 1 above)
Comment (mboersma) [inline _posts/2016-04-06-deis-1-13-lts.md]: I think the semicolon should be either a colon or comma.

> should be applied to [Deis Workflow](https://deis.com/), the successor to Deis.

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 3 above)
Comment (arschles) [inline _posts/2016-04-06-deis-1-13-lts.md]: Another option could be saying that `v1.13.0` is officially the long term release (LTS), and then you wouldn't have to put `LTS` everywhere else...

**Comment by bacongobbler** [comment]
Thanks guys! I'll make those changes.

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 3 above)
Comment (bacongobbler) [inline _posts/2016-04-06-deis-1-13-lts.md]: They are the same. We just tack on "LTS" as a marketing term to make it abundantly clear this is a long term support release.

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 1 above)
Comment (bacongobbler) [inline _posts/2016-04-06-deis-1-13-lts.md]: comma it is

File: _posts/2016-04-06-deis-1-13-lts.md (hunk 3 above)
Comment (bacongobbler) [inline _posts/2016-04-06-deis-1-13-lts.md]: @arschles sorry, running low on :tea:. I did exactly what you suggested :)

**Comment by bacongobbler** [comment]
I added all your changes except for the bold lettering on "Deis v1.13.0" and added "LTS" and "long term support" to the keyword metadata.

**Pull request merged**